components/spin/index.tsx

```diff
@@ -118,7 +121,7 @@ export default class Spin extends React.Component<SpinProps, any> {
         <Animate
           {...divProps}
           component="div"
-          className={`${prefixCls}-nested-loading`}
+          className={`${prefixCls}-nested-loading ${wrapperClassName}`}
```

Comment:
Ooops! This line:
```
const animateClassName = wrapperClassName ? prefixCls + '-nested-loading ' + wrapperClassName : prefixCls + '-nested-loading';
```
Produces: 
```
🚫 lint-staged:ts found some errors. Please fix them and try committing again.
/home/aaron/devel_global/ant-design/components/spin/index.tsx[116, 1]: Exceeds maximum line length of 120
```
It seems that I must avoid the use of the ternary conditional, but I think it's more clean use it:
```
    if (this.isNestedPattern()) {
      const animateClassName = wrapperClassName ? prefixCls + '-nested-loading ' + wrapperClassName : prefixCls + '-nested-loading';
      const containerClassName = classNames({
        [`${prefixCls}-container`]: true,
        [`${prefixCls}-blur`]: spinning,
      });
      return (
        <Animate
          {...divProps}
          component="div"
          className={animateClassName}
          style={null}
          transitionName="fade"
        >
          {spinning && <div key="loading">{spinElement}</div>}
          <div className={containerClassName} key="container">
            {this.props.children}
          </div>
        </Animate>
      );
    }
```
What do you think about it @afc163 & @benjycui ? I have two possibilities:
a) Ignore the warning running `git commit -n (or --no-verify)` 
b) Remove the ternary conditional and make a classic if/else conditional.

What do you prefer?

PS: As @benjycui said, snapshots have been returned to their original state.